git-hooks/post-checkout

```diff
@@ -1,3 +1,2 @@
-#!/usr/bin/env sh
-
-tar -C ./Carthage -xzf ./Carthage/Build.tar.gz
+#!/usr/bin/env bash
+exec git submodule update
```

Comment:
@aexmachina  i saw it in an example somewhere - possibly not.. i thought it might be something to do with paths (since another example i saw had to pass in the git_dir and other dirs as parameters). is it worth removing and re-testing?